entrypoint.py

```diff
@@ -0,0 +1,43 @@
+
+import argparse
+import json
+import os
+
+from safetab.create_tables import output_tables
+
+
+def make_tables(input_files, table_config):
+    
+    for input_file in input_files:
+        output_file_name = os.path.splitext(input_file)[0]
+        output_tables(data_csv=input_file,
+                      table_config=table_config,
+                      output_dir=f"{output_file_name}_tables")
+
+
+def main():
+    # make args parser
+    parser = argparse.ArgumentParser(description="Creates tables and redacts small numbers")
+
+    # version
+    parser.add_argument("--version", action='version', version="safetab 0.0.1")
+    
+    # configurations
+    parser.add_argument("config", type=str, help="The yaml passed in as a json obj")
+
+    # parse args
+    args = parser.parse_args()
+    
+    kwargs = vars(args)
+    yaml_str = kwargs.pop("config")
```

Comment:
This is not yaml, it's json, and it's not a string of json, it's a path to json data, so maybe `json_path` is a better variable name?

Reply:
agree - changed